lib/Refactoring/Async/AsyncRefactoring.h

```diff
@@ -274,7 +274,7 @@ enum class ConditionType {
   IS_TRUE,         // if b
   IS_FALSE,        // if !b
   SUCCESS_PATTERN, // case .success
-  FAILURE_PATTEN   // case .failure
+  FAILURE_PATTERN   // case .failure
```

Comment:
Could you keep the comments aligned?

```suggestion
  FAILURE_PATTERN  // case .failure
```